pages/common/ftp.md

```diff
@@ -0,0 +1,27 @@
+# ftp
+
+> Internet File Transfer Protocol
+
+- Connect to a FTP server:
+
+`ftp {{ftp.example.com}}`
+
+- Switch to binary transfer mode (graphics, compressed files etc):
+
+`binary`
+
+- Turn off interactive prompting for multiple file transfers:
```

Comment:
How about *Transfer multiple files without prompting for confirmation on every file:*? That way we're both more explicit, and we highlight the default behavior's silliness.